frame/support/src/weights.rs

```diff
@@ -179,25 +179,37 @@ pub enum SimpleDispatchInfo {
 	/// A normal dispatch with the maximum weight.
 	MaxNormal,
 	/// A normal dispatch with no weight.
-	FreeNormal,
+	///
+	/// WARNING: This causes a dispatchable call (i.e. transaction type) to be placed in the chain
+	/// an arbitrary number of times by any account (even a non-existence account) for no cost. IT
+	/// IS A DOS VECTOR. To use it correctly, there must be customised transaction validation logic
+	/// which limits inclusion of the transaction in some other way. If you do not do this, then
+	/// your chain WILL BE INSECURE.
+	InsecureFreeNormal,
```

Comment:
FreeNormal still have pays_fee = true so it is safe. i.e. it still needs the base fee + bytes fee. Only `FreeOperational` have pays_fee = false, which makes it insure.